src/main/java/mate/controller/AddDriverToCarController.java

```diff
@@ -0,0 +1,34 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Driver;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+@WebServlet(name = "chooseDriverToAdd", urlPatterns = {"/car/drivers/search/add"})
```

Comment:
Keep it `/cars` instead of `/car`, please read common mistakes file again and fix all the namings.